x-pack/plugins/synthetics/public/apps/synthetics/components/step_details_page/step_waterfall_chart/waterfall/sidebar.tsx

```diff
@@ -30,7 +30,7 @@ export const Sidebar: React.FC<SidebarProps> = ({ items, render }) => {
       >
         <EuiPanel css={{ height: '100%' }} hasBorder={false} hasShadow={false} paddingSize="none">
           <EuiFlexGroup
-            css={{ height: '100%' }}
+            css={{ height: '100%', paddingTop: 1.5 }}
```

Comment:
Correct me if I'm wrong because I don't know the specifics of the `css` prop but should the `paddingTop` value have a unit?

Reply:
i think it's fine in this case. Our sidebar was mis-aligned a bit from the waterfall bars. and this is the values that suited the most.